dff/context_storages/json.py

```diff
@@ -6,79 +6,173 @@
 store and retrieve context data.
 """
 import asyncio
-from typing import Hashable
+from pathlib import Path
+from base64 import encodebytes, decodebytes
+from typing import Any, List, Set, Tuple, Dict, Optional
+
+from pydantic import BaseModel, ConfigDict
```

Comment:
Pass config as class arguments for consistency with other modules
```suggestion
from pydantic import BaseModel
```